components/Staff/StaffMember.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Image from 'next/image'
+
+export default function StaffMember({ name, title, image }) {
+  return (
+    <div className="text-center mx-8 mb-4">
```

Comment:
on mobile, we would want 2 columns, 1 column is way to long. Also when you put in a long name, it messes with the alignment with the other profiles so you probably want to set like a max width for each staff member
![image](https://user-images.githubusercontent.com/62922309/184263734-2b7866a7-63a8-4215-8e87-d467bda154c4.png)